File: Solution in C++/Assignment_02/Task13.cpp
#include<iostream>
 using namespace std;
 int main(){
     int x;
     cin>>x;
     if(x<0 || x>100){cout<<"Wrong Input";}
     else if(x>=90){cout<<"A";}
     else if(x>=80){cout<<"B";}
     else if(x>=70){cout<<"C";}
     else if(x>=60){cout<<"D";}
     else if(x>=50){cout<<"E";}
     else{cout<<"F";}
 }

File: Solution by Python/Assignment_02/Task13.py
n=int(input())
if n<0 or n>100:
    print("Invalid")
elif n>=90:
    print("A")
elif n>=80:
    print("B")
elif n>=70:
    print("C")
elif n>=60:
    print("D")
elif n>=50:
    print("E")
else:
    print("F")